### ISSUE TITLE
SpecialKey stands out way too much

### ISSUE THREAD

**Issue by vdrandom** [freeo/vim-kalisi#7]
It is especially annoying for those of us who keep 'list' option always on to distinguish tabs. Toning it down a bit would help, or even implementing a variable to lower the contrast, like in solarized or gruvbox colorschemes.

Here's an example of making it less annoying:
```diff
diff --git a/colors/kalisi.vim b/colors/kalisi.vim
index 730567c..f451784 100644
--- a/colors/kalisi.vim
+++ b/colors/kalisi.vim
@@ -250,7 +250,7 @@ if &background == "dark"
     hi Typedef ctermfg=72
     hi Special ctermfg=194 cterm=none
     hi SpecialChar ctermfg=69 cterm=none
-    hi SpecialKey ctermbg=236 ctermfg=194 cterm=bold
+    hi SpecialKey ctermbg=238 ctermfg=240
     hi Tag cterm=bold ctermfg=39
     hi Delimiter ctermfg=104
     hi SpecialComment cterm=bold ctermfg=67
```
Screenshots:
before patch - http://i.imgur.com/6IJJGPe.png
after patch - http://i.imgur.com/rTfDcTo.png

**Comment by mkrull** [comment]
If the background color should be the same as the default background just leave it out entirely.

Anyway is the high visibility intentional?

**Comment by freeo** [comment]
Yes, the high visibility was intentional because I thought SpecialKey was only special Keys like `>M` etc. Now that I've looked into the `list` and `listchars` option I decided to tone it down. A lot. 

Fixed in 0.8 (yet unreleased).
